client/src/js/reducers/documentReducers.js

```diff
@@ -0,0 +1,42 @@
+import actionTypes from '../constants/actionTypes';
+
+const initialState = { documents: [], userDocuments: [] };
+
+const documentReducer = (state = initialState, action) => {
+  switch (action.type) {
+  case actionTypes.CREATE_DOCUMENT_SUCCESS: {
+    const editID = action.createdDocument.id;
+    const stateUserDocuments = state.userDocuments;
+    return { ...state,
+      userDocuments: stateUserDocuments.push(action.createdDocument),
```

Comment:
I believe that you are mutating the state at this point, you should avoid state mutation when using redux.